setup.cfg

```diff
@@ -0,0 +1,52 @@
+[metadata]
+name = openstack-tests-list
+summary = Tool to manage Tempest test skiplists and allowlists
+description_file =
+    README.rst
+author = Soniya Vyas
+author_email = svyas@redhat.com
+home_page = https://github.com/openstack-k8s-operators/openstack-tests-list
+classifier =
+    Environment :: OpenStack
+    Intended Audience :: Information Technology
+    Intended Audience :: System Administrators
+    License :: OSI Approved :: Apache Software License
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
what about the newer ones? right now the upstream started testing with python3.12